os/board/common/partitions.c

```diff
@@ -91,15 +91,20 @@ void configure_partitions(void)
 			lldbg("ERROR: failed to create partition.\n");
 			return;
 		}
-#if defined(CONFIG_MTD_FTL)
-		if (!strncmp(types, "ftl,", 4)) {
+#ifdef CONFIG_MTD_FTL
```

Comment:
RomFS can be mounted based on ram mtd, please consider rammtd too.

Reply:
Thanks. I will check.